src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -592,6 +592,7 @@ private static bool IsStandardImplicitConversionFromExpression(ConversionKind ki
             // to include conversions from expression.
             switch (kind)
             {
+                case ConversionKind.NullLiteral:
```

Comment:
Should we also add the `DefaultLiteral` conversion?

Reply:
This method should return `true` only for "standard implicit conversions", and `DefaultLiteral` doesn't seem to be in [the list](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/conversions.md#1042-standard-implicit-conversions).